lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -444,6 +444,48 @@ export class EvaluationClientWrapper {
     return result;
   }
 
+  private extractWrappedVcFromWrappedVp(
+    descriptor: Descriptor,
+    descriptorIndex: string,
+    wvp: WrappedVerifiablePresentation,
+  ): { error: Checked; wvc: undefined } | { wvc: WrappedVerifiableCredential; error: undefined } {
+    // Decoded won't work for sd-jwt or jwt?!?!
```

Comment:
What do you mean with this comment? Decoded should have a value that would work when there would be JSON-LD credentials right? Also I need to double check whether we would also decode VCs when decoding VPs

Reply:
Not relevant anymore. For sd-jwt this won't work but this code will never be used when sd-jwt is used (as submission is always external). I'll remove the comment